### PR TITLE
Packit/RPM: Initial add of config and rpm spec

### PR DESCRIPTION
**Pull request by lsm5**
Include validate and unit tests using TMT.
    
Also cleanup a leftover double quote in test.sh from previous commit.

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    targets:
+      fedora-all-x86_64: {}
+      fedora-all-aarch64: {}
+      fedora-eln-x86_64:
+        additional_repos:
+          - "https://kojipkgs.fedoraproject.org/repos/eln-build/latest/x86_64/"
+      fedora-eln-aarch64:
+        additional_repos:
+          - "https://kojipkgs.fedoraproject.org/repos/eln-build/latest/aarch64/"
+      epel-8-x86_64: {}
```
Comment (rhatdan) [inline .packit.yaml:28]: I don't think we should support RHEL8 only 9 and later

**Comment by lsm5** [comment]
rust on rhel9 is 1.71 while crun-vm needs 1.74. Disabling epel build targets for now.

**Comment by lsm5** [comment]
Good to go. rpm built in copr installs `/usr/bin/crun-vm`. Once this is merged, crun-vm will be built automatically on `rhcontainerbot/podman-next` after each commit to main. Might wanna review and merge #14 first for Makefile.

**Comment by lsm5** [comment]
Also, might wanna consider doing a 0.0.1 release or whatever to get official Fedora packaging going. Not a requirement but could be nice to have instead of just building a commit id.

File: rpm/crun-vm.spec [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+%endif
+# DO NOT TOUCH the Version string!
+# The TRUE source of this specfile is:
+# https://github.com/containers/crun-vm/blob/main/rpm/crun-vm.spec
+# If that's what you're reading, Version must be 0, and will be updated by Packit for
+# copr and koji builds.
+# If you're reading this on dist-git, the version is automatically filled in by Packit.
+Version: 0
+# The `AND` needs to be uppercase in the License for SPDX compatibility
+License: Apache-2.0 AND MIT AND Zlib
```
Comment (albertofaria) [inline rpm/crun-vm.spec:37]: This probably has to be: `(Apache-2.0 OR BSL-1.0) AND (Apache-2.0 OR MIT) AND (MIT OR Apache-2.0) AND (MIT OR Unlicense) AND Apache-2.0 AND GPL-2.0-or-later AND MIT AND MPL-2.0 AND MPL-2.0+ AND Unicode-DFS-2016`

File: .packit.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rpm/crun-vm.spec
+upstream_tag_template: v{version}
```
Comment (albertofaria) [inline .packit.yaml:6]: I cut a 0.1.0 release now. The tag doesn't have a `v` prefix.

File: rpm/crun-vm.spec [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+Release: %autorelease
+%if %{defined golang_arches_future}
+ExclusiveArch: %{golang_arches_future}
+%else
+ExclusiveArch: aarch64 ppc64le s390x x86_64
+%endif
+Summary: OCI Runtime to run QEMU-compatible VM images
+URL: https://github.com/containers/%{name}
+# Tarballs fetched from upstream's release page
+Source0: %{url}/archive/v%{version}.tar.gz
```
Comment (albertofaria) [inline rpm/crun-vm.spec:47]: Also need to drop the `v` prefix here.

**Review by albertofaria — CHANGES_REQUESTED** [review]

**Comment by rhatdan** [comment]
@lsm5 Needs a rebase.

**Comment by lsm5** [comment]
Added unit test runs on each target using TMT. lmk if it's overkill.

**Comment by lsm5** [comment]
@albertofaria @rhatdan PTAL

**Review by albertofaria — APPROVED** [review]

**Comment by albertofaria** [comment]
Thanks, LGTM.

**Comment by lsm5** [comment]
crun-vm is now installable via `rhcontainerbot/podman-next` copr.

**Comment by rhatdan** [comment]
We are off to the races.
Now to get it into Fedora...

**Pull request merged**